sbt-scalajs-bundler/src/main/scala-sbt-0.13/scalajsbundler/Compat.scala

```diff
@@ -0,0 +1,12 @@
+package scalajsbundler
+
+object Compat {
+  type Process = sbt.Process
+  val Process = sbt.Process
+  type ProcessLogger = sbt.ProcessLogger
+
+  // Mimics scala.sys.process.ProcessLogger
+  implicit class ProcessLoggerOps(val self: ProcessLogger) extends AnyVal {
+    def out(msg: String) = self.info(msg)
+  }
+}
```

Comment:
Why not use `scala.sys.Process` everywhere instead of introducing this abstraction over `scala.sys.Process` and `sbt.Process`?

Reply:
I will give it a try.

Reply:
Done in 8f2e318